os/adding-swap.md

```diff
@@ -0,0 +1,74 @@
+# Adding swap in CoreOS
+
+Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
+
+By default CoreOS does not include a partition for swap, however one can configure their system to have swap.
+
+## Manually adding a swapfile on ext4
+
+Run these commands **as root** to create a 512 megabyte swapfile.
+
+```
+$ fallocate -l 512m /media/state/512MiB.swap
+$ chmod 600 /media/state/512MiB.swap
+$ chown root /media/state/
+$ mkswap /media/state/512MiB.swap
+```
+
+**Note**: The partition you are creating your swapfile on must meet the following requirements:
+
+* Must not be of type `tmpfs`.
+* Must be large enough for your swapfile size.
+
+To verify that our swapfile meets those requirements, run the following command:
+
+```
+$ df -Th
+Filesystem     Type      Size  Used Avail Use% Mounted on
+[...]
+/dev/sdXN      ext4      2.0G  3.0M  1.8G   1% /media/state
+```
+
+Our block device `/dev/sdXN` is not `tmpfs` and has enough space for a 512MiB swapfile, so we can proceed.
+
+### Create a unit to initialize the swap on boot
+
+Create a unit file at `/media/state/units/swapon.service` with these contents:
```

Comment:
`/etc/systemd/system/swapon.service`.

Can you provide an Ignition config that does this?